docs/installation.rst

```diff
@@ -0,0 +1,201 @@
+.. meta::
+   :description: A high-performance HIP library for tensor primitives
+   :keywords: hipTensor, ROCm, library, API, tool, installation
+
+.. _installation:
+
+===============================
+Installation
+===============================
+
+This document provides instructions for installing and configuring hipTensor library on Linux.
+The quickest way to install is to build from source.
+
+-------------
+Prerequisites
+-------------
+
+-  A ROCm enabled platform, more information `here <https://github.com/ROCm/ROCm>`_.
+-  ROCm-cmake, more information `here <https://github.com/RadeonOpenCompute/rocm-cmake/>`_.
+
+---------------------------------
+Building and installing hipTensor
+---------------------------------
+
+Use the following instructions to build hipTensor from source.
+
+System requirements
+^^^^^^^^^^^^^^^^^^^
+As a general rule, 8GB of system memory is required for a full hipTensor build. This value can be lower if hipTensor is built without tests. This value may also increase in the future as more functions are added.
+
+GPU support
+^^^^^^^^^^^
+AMD CDNA class GPU featuring matrix core support: `gfx908`, `gfx90a`, `gfx940`, `gfx941`, `gfx942` and `gfx9`.
```

Comment:
gfx9 is an umbrella term for the listed gfx*** arches. 

Perhaps can update to "gfx908, gfx90a, ... , gfx942" as 'gfx9'